chairing.py

```diff
@@ -19,43 +17,64 @@
-    "The speakers in your session should also be present before the start of the session to introduce themselves. <b>Please confirm the correct pronunciation of their names</b>, especially if the name is unfamiliar to you.",
-    "In case you need it, the username for the PC is <b>{login_username}</b>, and the password is <b>{login_password}</b>.",
+    "Your session starts at <b>{start_time} on {day}</b>. Please arrive at room <b>{room}</b> 10 minutes in advance of your session. Your room will have a volunteer and stream director present to help the event run smoothly—they will manage such things as moderating the Slido questions, ensuring speakers have microphones, loading up presentations onto the PC to display, and ensuring remote presenters are connected to the Zoom call and audible. Please introduce yourself to your volunteer and stream director before the start of the session.",
+    "The speakers in your session should also be present before the start of the session to introduce themselves. <b>Please confirm the correct pronunciation of their names and pronouns</b>, especially if the name is unfamiliar to you.",
+    "In case you need it, the username for the PC in the room is <b>{login_username}</b>, and the password is <b>{login_password}</b>, but again, your room volunteer should be able to handle anything with this machine. Ensure that the volunteer or stream director has the Slido wall view on one of the projected screens. The direct link <a href={slido_room_wall_link}><u>is here</u></a>. Alternatively, navigate to <a href={slido_event_wall}><u>{slido_event_wall}</u></a> and select the room {room}. It might drop you into the wrong room by default. Click next to the coloured dot at the top with text starting 'GH' to get a drop down of rooms.",
     ("Before each talk", styles["Heading1"]),
-    "Please briefly introduce the speaker—a full bio is not needed, just the presenter’s name and the title of the talk. Remind the audience that they can ask questions via Sli.do.",
+    "Please briefly introduce the speaker—a full bio is not needed, just the presenter's name and the title of the talk. Only do this after the buffer time between talks has elapsed and you have confirmed that the speaker is ready to start.",
+    "Remind the audience that they can ask questions via Slido and point to the Slido wall slide. The manual procedure if not using the QR code is to navigate to <a href=https://slido.com>slido.com</a> and enter the event code: {slido_event_code}. Audience members will then need to select the correct room from the drop down. Remind them that it is {room}.",
```

Comment:
Does the `<a>` tag work correcty in ReportLab? I'd at least expect some quotes around the URL, but I haven't run the code to test the output explicitly.